frontend/server/src/Controllers/QualityNomination.php

```diff
@@ -991,9 +1001,17 @@ private static function validateMemberOfReviewerGroup(\OmegaUp\Request $r) {
      *
      * @template T
      * @param array<T> $contents
+     * @return array{duplicates: string}
      */
-    private static function hasDuplicates(array $contents): bool {
-        return count($contents) !== count(array_unique($contents));
+    private static function hasDuplicates(array $contents): array {
+        $counts = array_count_values($lista);
+        $duplicates = array();
```

Comment:
Desde la versión de PHP 5.4 y posteriores ya existe una forma abreviada de inicializar un array
```suggestion
        $duplicates = [];
```
¿te parece si lo usamos mejor de esa forma?